pkg/catalog/catalog_test.go

```diff
@@ -67,6 +67,9 @@ func TestVersionExist(t *testing.T) {
 				return true, testingutil.FakeImageStreams(tt.args.name, tt.args.namespace, tt.args.tags), nil
 			})
 
+			fakeClientSet.ImageClientset.PrependReactor("list", "imagestreams", func(action ktesting.Action) (bool, runtime.Object, error) {
+				return true, testingutil.FakeImageStreams(tt.args.name, "openshift", tt.args.tags), nil
+			})
```

Comment:
Is there a reason why we have two `PrependReactor`s defined here? Can't this be made generic?

Reply:
I am really not sure :)
May be added by mistake..
Removing it